1034-toggle-design-guidance.md.md

```diff
@@ -0,0 +1,22 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-18
+updated: 2024-10-22
+---
+
+# Toggle switch Component
+
+[Toggle switch](https://clarity.design/documentation/toggle-switch) component allows users to switch between two mutually exclusive options while always having a default value.
+
+## Guidance
+
+- **Must** use for actions that occur immediately after the user press to on/off certain feature without an additional submit or save action.
```

Comment:
Design teams **Must** use this pattern for actions that occur immediately after the user presses to turn a feature on or off, without requiring an additional submit or save action. This ensures a seamless and intuitive user experience.